src/fourcipp/fourc_input.py

```diff
@@ -448,7 +453,7 @@ def dump(
         if convert_to_native_types:
             self.convert_to_native_types()
 
-        dump_yaml(self.inlined, input_file_path, sort_sections)
+        dump_yaml(self.inlined, input_file_path, self.sort_function)
```

Comment:
From the comment above:
```suggestion
        dump_yaml(self.inlined, input_file_path, sort_function)
```